files/codeclimate/.make_codeclimate

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
```

Comment:
I like this pattern of combining the pieces necessary for the languages. Since many of the languages in the default .codeclimate.yml are used for a small number of repos, do you think it makes sense to eventually move them to the same model? (Not suggesting it should be done as part of this PR, but thinking about future work).

Reply:
Exactly, that is the plan.